test/config-luatest/labels_test.lua

```diff
@@ -0,0 +1,35 @@
+local t = require('luatest')
+local helpers = require('test.config-luatest.helpers')
+local cbuilder = require('test.config-luatest.cbuilder')
+local cluster = require('test.config-luatest.cluster')
+
+local g = helpers.group()
```

Comment:
Nit: It performs all the job regarding stopping tarantool processes and removing temporary files/directories. However, it leaves `g.cluster` in an inconsistent state.

For example, an attempt to add another test case that uses `cluster.new()` would fail. (At least I guess that it is so based on the source code.)

Please, use the usual boilerplate.

https://github.com/tarantool/tarantool/blob/5040fba9cf1da942371721e36e81c7372699600c/test/config-luatest/cluster.lua#L3-L12

`helpers.group()` fits good, when functions from this helper module are in use.

Reply:
Thanks for the catch. Fixed.

Reply:
It calls the treegen's `init`/`clean` functions twice. It is harmless, but I would simplify it a bit. In fact, we don't need the `test/config-luatest/helpers.lua` usage at all.

```diff
diff --git a/test/config-luatest/labels_test.lua b/test/config-luatest/labels_test.lua
index 0db0235c27..5da961f2cc 100644
--- a/test/config-luatest/labels_test.lua
+++ b/test/config-luatest/labels_test.lua
@@ -1,9 +1,8 @@
 local t = require('luatest')
-local helpers = require('test.config-luatest.helpers')
 local cbuilder = require('test.config-luatest.cbuilder')
 local cluster = require('test.config-luatest.cluster')
 
-local g = helpers.group()
+local g = t.group()
 
 g.before_all(cluster.init)
 g.after_each(cluster.drop)
```

I'm going to just fix it and don't play a ping-pong.

I want to eliminate a need in this boilerplate during integration of the helpers into `luatest`. When `treegen` and `cluster` will be part of `luatest`, these modules should automatically add appropriate hooks.

Reply:
Force-pushed the branch.